src/Pool.sol

```diff
@@ -327,21 +312,18 @@ abstract contract Pool {
 
         // Dripped when collected
         if (collected > 0 && currReceivers.length > 0 && sender.dripsFraction > 0) {
-            uint256 drippable = (uint256(collected) * sender.dripsFraction) / DRIPS_FRACTION_MAX;
-            uint32 weightsSum = _weightsSum(currReceivers);
-            dripped = uint128(drippable - (drippable % weightsSum));
-            collected -= dripped;
-            uint128 drippedPerWeight = dripped / weightsSum;
-            uint128 actuallyDripped = 0;
+            uint256 drippable = (uint256(collected) * sender.dripsFraction) / MAX_DRIPS_FRACTION;
+            uint128 totalAmtPerSec = _totalAmtPerSec(currReceivers);
```

Comment:
Same here: `totalAmtPerSec` could be provided as parameter and afterwards calc the acutal one and compare both.